src/configuration.cpp

```diff
@@ -451,20 +453,36 @@ chimera::CompiledConfiguration::CompiledConfiguration(
     // Override individual templates if specified in the configuration.
     if (bindingNode_)
     {
-        if (const YAML::Node classTemplateNode = bindingNode_["class"])
-            bindingDefinition_.class_cpp = Lookup(classTemplateNode);
+        if (const auto node = lookupYAMLNode(bindingNode_, "class", "header"))
+            bindingDefinition_.class_h = Lookup(node);
+
+        if (const auto node = lookupYAMLNode(bindingNode_, "class", "source"))
+            bindingDefinition_.class_cpp = Lookup(node);
+
+        if (const auto node = lookupYAMLNode(bindingNode_, "enum", "header"))
+            bindingDefinition_.enum_h = Lookup(node);
+
+        if (const auto node = lookupYAMLNode(bindingNode_, "enum", "source"))
+            bindingDefinition_.enum_cpp = Lookup(node);
 
-        if (const YAML::Node enumTemplateNode = bindingNode_["enum"])
-            bindingDefinition_.enum_cpp = Lookup(enumTemplateNode);
+        if (const auto node
+            = lookupYAMLNode(bindingNode_, "function", "header"))
+            bindingDefinition_.function_h = Lookup(node);
 
-        if (const YAML::Node functionTemplateNode = bindingNode_["function"])
-            bindingDefinition_.function_cpp = Lookup(functionTemplateNode);
+        if (const auto node
+            = lookupYAMLNode(bindingNode_, "function", "source"))
+            bindingDefinition_.function_cpp = Lookup(node);
 
-        if (const YAML::Node moduleTemplateNode = bindingNode_["module"])
-            bindingDefinition_.module_cpp = Lookup(moduleTemplateNode);
+        if (const auto node = bindingNode_["module"])
+            bindingDefinition_.module_cpp = Lookup(node);
```

Comment:
Let's use `lookupYAMLNode()` here as well for consistency.
```suggestion
        if (const auto node = lookupYAMLNode(bindingNode_, "module"))
            bindingDefinition_.module_cpp = Lookup(node);
```